app/views/checkouts/index.html.erb

```diff
@@ -34,14 +34,14 @@
 
     <div class="d-none d-md-flex row font-weight-bold list-header">
       <div class="row col-md-4">
-        <div class="col-md-6" data-sort="status">Status</div>
-        <div class="col-md-6 active" data-sort="due_date">Due date</div>
+        <div class="col-md-12 col-lg-6" data-sort="status">Status</div>
+        <div class="col-md-12 col-lg-6 active" data-sort="due_date">Due date</div>
       </div>
       <div class="row col-md-8">
         <div class="col-md-7" data-sort="title">Title</div>
         <div class="row col-md-5">
-          <div class="col-md-6" data-sort="author">Author</div>
```

Comment:
Do we want to do this same collapse behavior for requests + fines?

Reply:
Great question. @jvine Do you want the same behavior there?

Reply:
I've popped on a commit that does it in case we do.

Reply:
Yes, we do. (Royal we.)

Reply:
Great.. I think I've made the changes needed here now